### PR TITLE
Added Rails 5.2 upgrade article

### PR DESCRIPTION
**Pull request by mauro-oto**
Hey!

This PR adds the Rails 5.1 to 5.2 upgrade article. 

Please check it out, thanks!

### CODE REVIEW

File: _posts/2018-06-19-upgrade-rails-from-5-1-to-5-2.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+and other gems, like [Shrine](https://github.com/shrinerb/shrine). Paperclip in
+particular [has been deprecated](https://github.com/thoughtbot/paperclip#deprecated).
+If you're using Paperclip, the maintainers wrote an in-depth migration guide
+which you can [follow here](https://github.com/thoughtbot/paperclip/blob/master/MIGRATING.md).
+
+<h3 id="credentials">b. Credentials</h2>
+
+Another major feature is the introduction of Credentials. It will eventually
+replace the current `config/secrets.yml`. Since there are also encrypted
+secrets (`config/secrets.yml.enc`), its intention is to clear up the confusion.
```
Comment (ercohen14) [inline _posts/2018-06-19-upgrade-rails-from-5-1-to-5-2.markdown]: it's intention (add a comma)

File: _posts/2018-06-19-upgrade-rails-from-5-1-to-5-2.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+For more information on Credentials and a work-around to this last issue, check
+out [this post](https://blog.eq8.eu/til/rails-52-credentials-tricks.html), where
+they use `Hash#dig` to configure per-environment credentials.
+
+<h2 id="next-steps">5. Next steps</h2>
+
+If you successfully followed all of these steps, you should now be running Rails 5.2! Do you have any other useful tips or recommendations? Did we miss anything? Share them with us in the comments section.
+
+If you don't have the time to upgrade your Rails app, check out our [Ruby on Rails
+upgrade](https://fastruby.io) service: [FastRuby.io](https://fastruby.io)
```
Comment (ercohen14) [inline _posts/2018-06-19-upgrade-rails-from-5-1-to-5-2.markdown]: I think just the link on FastRuby.io is enough. Maybe you don't need it for "Ruby on Rails upgrade"

File: _posts/2018-07-26-upgrade-rails-from-5-1-to-5-2.markdown [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+[this post](http://thomasleecopeland.com/2015/08/06/running-rails-update.html).
+
+As an alternative, check out [RailsDiff](http://railsdiff.org/5.1.6/5.2.0),
+which provides an overview of the changes in a basic Rails app between 5.1.x and
+5.2.x (or any other source/target versions).
+
+<h2 id="application-code">4. Application code</h2>
+
+There were not too many deprecations between 5.1 and 5.2 compared to past
+releases ([see the full changelog here](https://guides.rubyonrails.org/5_2_release_notes.html#upgrading-to-rails-5-2)).
```
Comment (etagwerker) [inline _posts/2018-07-26-upgrade-rails-from-5-1-to-5-2.markdown]: @mauro-oto maybe you can mention one or two deprecation warnings?

Comment (mauro-oto) [reply _posts/2018-07-26-upgrade-rails-from-5-1-to-5-2.markdown]: @etagwerker I thought about it but I don't think that's necessary, deprecations like `Deprecate after_bundle callback in Rails plugin templates` and `Deprecate Module#reachable? method` are not very relevant to most Rails apps.

**Pull request merged**